src/services/config.js

```diff
@@ -30,5 +30,6 @@ if (admin_pass) {
 
 config.auth = {
   basic_enabled: Config.castBool(config.getEnv('AUTH_BASIC_ENABLED'), true),
-  http_signature_enabled: Config.castBool(config.getEnv('AUTH_HTTP_SIGNATURE_ENABLED'), true)
+  http_signature_enabled: Config.castBool(config.getEnv('AUTH_HTTP_SIGNATURE_ENABLED'), true),
+  client_certificates_enabled: Config.castBool(config.getEnv('CLIENT_CERTS_ENABLED'), true)
```

Comment:
Since it doesn't look like a default .pfx is being bundled, shouldn't `client_certificates_enabled` be disabled by default?